routes/comments.js

```diff
@@ -1,9 +1,12 @@
+const campground = require("../models/campground");
+const { route } = require("./campgrounds");
+
 const   express = require("express"),
-        router = express.Router(),
+        router = express.Router({mergeParams: true}),
```

Comment:
What does that do? the `mergeParams: true`

Reply:
With this we merge the params of the comments with the campgrounds params and we are able to access the campground id in the comments route.